src/exercises/exercise09/Dogs.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Flex, Box } from 'grid-styled';
+import DogPhoto from '../../components/app/DogPhoto';
+
+const Dogs = ({ dogs }) => (
+	<div>{dogs.map(dog => <DogPhoto id={dog.id} breed={dog.name} />)}</div>
```

Comment:
Is this a completed eyxcersice? Or what's the goal of the exercise?

Reply:
Check the readme ;-)

Reply:
Ah gotcha.